CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Fixes
 
-- Fix issue with invalid profiles uploading (#2357)
+- Fix issue with invalid profiles uploading (#2357 and #2358)
```

Comment:
m: I think this should be removed. 2357 was closed and added nothing. 2358 has its own changelog entry.

```suggestion
```

Reply:
fixed the numbers on these